app-vcs/brz/autobuild/prepare

```diff
@@ -0,0 +1 @@
+sed -i 's|man/man1|share/man/man1|' setup.py
```

Comment:
Do not use sed as its results are not controlled. Please generate a patch with this command.